packaging/debian/postinst

```diff
@@ -25,31 +25,82 @@ DAEMON_USER=storj-storagenode
 
 . /usr/share/debconf/confmodule
 
+arch=$(dpkg --print-architecture)
+
+get_binary() {
+    binary=$1
+    tmpfile=$(mktemp /tmp/${binary}.zip.XXXXXX)
+
+    wget -O $tmpfile ${BINARIES_SERVER}/${binary}_linux_${arch}.zip
+    unzip $tmpfile -d /var/lib/storagenode
+    mv /var/lib/storagenode/${binary}_linux_amd64 /var/lib/storagenode/${binary}
+    rm $tmpfile
+}
 
 case "$1" in
     configure)
```

Comment:
This line should be `configure|abort-upgrade|abort-remove)`

That is, all of these actions should be taken when:

 * we are installing the package for the first time
 * we are upgrading the package and about start things up again
 * something went wrong when trying to upgrade this package to a newer version, and the user wants the old version put back in working order
 * something went wrong when trying to remove the package, and the user wants it back in working order